### PR TITLE
feat: add some nepali commands

### PR DESCRIPTION
**Pull request by thenishantsapkota**

### CODE REVIEW

File: src/commands/General/CovidCommand.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                    value: data.deaths.toString(),
+                    inline: true
+                },
+                {
+                    name: "Recovered",
+                    value: data.recovered.toString(),
+                    inline: true
+                },
+                {
+                    name: "Today's Cases",
```
Comment (DevAndromeda) [inline src/commands/General/CovidCommand.ts:59]: `name: "Cases Today",`

File: src/commands/General/CovidCommand.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                    value: data.todayCases.toString(),
+                    inline: true
+                },
+                {
+                    name: "Today's Deaths",
```
Comment (DevAndromeda) [inline src/commands/General/CovidCommand.ts:64]: `name: "Deaths Today",`

File: src/commands/General/CovidCommand.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
+import { Client, CommandInteraction, MessageEmbed, MessageActionRow, MessageButton } from "discord.js";
+import { inject, injectable } from "tsyringe";
+import fetch from "node-fetch";
+
+import BaseCommand from "#base/BaseCommand";
+import { kClient } from "#utils/tokens";
+
+interface APIResponse{
```
Comment (DevAndromeda) [inline src/commands/General/CovidCommand.ts:8]:
```ts
interface APIResponse{
    country: string;
    cases: number;
    todayCases: number;
    deaths: number;
    todayDeaths: number;
    recovered: number;
    active: number;
    critical: number;
    totalTests: number;
}
```

**Review by DevAndromeda — CHANGES_REQUESTED** [review]

**Review by TheLearneer — APPROVED** [review]
LGTM

**Pull request merged**